### ISSUE TITLE
STP - lot of consuming data bytes errors - not enough info in logs

### ISSUE THREAD

**Issue by yanpom** [thomsonreuters/CM-Well#571]
Lot of similar errors in agents.log during STP work - not enough info - it says that response body is not valid - Why its not valid - how do I know it succeed on next retry? Just because there is no additional error?
response body with proper error need to be printed to tools-red.log

```
agents.log:2018-03-07 07:02:07,421 [scala-execution-context-global-265]         ERROR cmwell.tools.data.utils.akka.Retry$  - [oafulloa-relationship] error in consuming data bytes from response, paths=/data.thomsonreuters.com/2-0ebab5f625a6fa60bf3649240cecd4ab6d0aae74b78ac6b6aaef3ec64feff5c9, err=akka.stream.scaladsl.TcpIdleTimeoutException: TCP idle-timeout encountered on connection to [cm-well-ppe.int.thomsonreuters.com:80], no bytes passed in the last 1 minute
agents.log:2018-03-07 07:02:07,428 m-well-ppe-akka.actor.default-dispatcher-43] WARN  cmwell.tools.data.utils.akka.Retry$  - [oafulloa-relationship] received 200 OK but response body is not valid, will retry again in 30 seconds host=c285msfcsnd07.int.thomsonreuters.com data=/data.thomsonreuters.com/2-0ebab5f625a6fa60bf3649240cecd4ab6d0aae74b78ac6b6aaef3ec64feff5c9
```

**Comment by nicklester** [comment]
New response headers from _sp API with associated logging solved in #434 and #578 #503 should resolve this. Please can you confirm?

**Comment by yanpom** [comment]
Verified and Closed
